lib/PuppeteerSharp/Messaging/PageJavascriptDialogOpeningMessageResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace PuppeteerSharp.Messaging
```

Comment:
@Meir017 I'm introducing this `Messaging` namespace so we can start working on #97, little by little